src/Pretty.hs

```diff
@@ -415,7 +428,7 @@ instance (Pretty VarInfo s, Pretty VarInfo t, PrettyBinder VarInfo a, Pretty Var
   ppBind env (Example loc t e) =
     (hang 4 $
      text "example@" <> pp env loc <+>
-     align (group (vsep [ group (pp env e) <+> text ":"
```

Comment:
This seems definitely to be a regression - surely the two sides of the type ascription should avoid breaking if a break on the colon fixes it?